### PR TITLE
Add missing clang versions for armv7-a C++

### PR DESCRIPTION
**Pull request by DavidSpickett**
These were present for C but not C++.

### CODE REVIEW

File: etc/config/c++.amazon.properties [hunk 1]
```diff
@@ -643,6 +643,33 @@ group.armclang32.licenseLink=https://github.com/llvm/llvm-project/blob/main/LICE
...
+compiler.armv7-clang1400.exe=/opt/compiler-explorer/clang-14.0.0/bin/clang
+compiler.armv7-clang1400.semver=14.0.0
+compiler.armv7-clang1400.options=-target arm-linux-gnueabi --gcc-toolchain=/opt/compiler-explorer/arm/gcc-11.2.0/arm-unknown-linux-gnueabihf --sysroot=/opt/compiler-explorer/arm/gcc-11.2.0/arm-unknown-linux-gnueabihf/arm-unknown-linux-gnueabihf/sysroot
+compiler.armv7-clang1301.exe=/opt/compiler-explorer/clang-13.0.1/bin/clang
+compiler.armv7-clang1301.semver=13.0.1
+compiler.armv7-clang1301.options=-target arm-linux-gnueabi --gcc-toolchain=/opt/compiler-explorer/arm/gcc-11.2.0/arm-unknown-linux-gnueabihf --sysroot=/opt/compiler-explorer/arm/gcc-11.2.0/arm-unknown-linux-gnueabihf/arm-unknown-linux-gnueabihf/sysroot
+compiler.armv7-clang1300.exe=/opt/compiler-explorer/clang-13.0.0/bin/clang
+compiler.armv7-clang1300.semver=13.0.0
+compiler.armv7-clang1300.options=-target arm-linux-gnueabi --gcc-toolchain=/opt/compiler-explorer/arm/gcc-11.2.0/arm-unknown-linux-gnueabihf --sysroot=/opt/compiler-explorer/arm/gcc-11.2.0/arm-unknown-linux-gnueabihf/arm-unknown-linux-gnueabihf/sysroot
```
Comment (RubenRBS) [inline etc/config/c++.amazon.properties:666]: Am I missunderstanding something, or shouldnt these be the clang++ variants?

**Review by RubenRBS — COMMENTED** [review]
Thanks a lot for taking the time to add the missing compilers, we appreciate it! :)

File: etc/config/c++.amazon.properties (hunk 1 above)
Comment (DavidSpickett) [reply etc/config/c++.amazon.properties:666]: They should be yes, I've fixed that.

**Review by RubenRBS — APPROVED** [review]

**Comment by RubenRBS** [comment]
Thanks a lot! :)

**Comment by dkm** [comment]
This is live :rocket:

**Comment by dkm** [comment]
This is live :rocket:

**Pull request merged**